src/test/regress/expected/mixed_relkind_tests.out

```diff
@@ -0,0 +1,932 @@
+                                                 QUERY PLAN
+---------------------------------------------------------------------
+ Custom Scan (Citus INSERT ... SELECT)
+   INSERT/SELECT method: repartition
+   ->  Custom Scan (Citus Adaptive)
+         Task Count: 4
+         Tasks Shown: One of 4
+         ->  Task
+               Node: host=localhost port=xxxxx dbname=regression
+               ->  Append
+                     ->  Seq Scan on partitioned_distributed_table_1_1513014 partitioned_distributed_table_1
+                     ->  Seq Scan on partitioned_distributed_table_2_1513018 partitioned_distributed_table_2
+(10 rows)
+
+EXPLAIN (COSTS OFF) INSERT INTO foreign_distributed_table SELECT a + 1 FROM partitioned_distributed_table;
+                                                 QUERY PLAN
+---------------------------------------------------------------------
+ Custom Scan (Citus INSERT ... SELECT)
+   INSERT/SELECT method: repartition
+   ->  Custom Scan (Citus Adaptive)
+         Task Count: 4
+         Tasks Shown: One of 4
+         ->  Task
+               Node: host=localhost port=xxxxx dbname=regression
+               ->  Append
+                     ->  Seq Scan on partitioned_distributed_table_1_1513014 partitioned_distributed_table_1
+                     ->  Seq Scan on partitioned_distributed_table_2_1513018 partitioned_distributed_table_2
+(10 rows)
+
+EXPLAIN (COSTS OFF) INSERT INTO partitioned_distributed_table SELECT a + 1 FROM distributed_table;
+                                QUERY PLAN
+---------------------------------------------------------------------
+ Custom Scan (Citus INSERT ... SELECT)
+   INSERT/SELECT method: repartition
+   ->  Custom Scan (Citus Adaptive)
+         Task Count: 4
+         Tasks Shown: One of 4
+         ->  Task
+               Node: host=localhost port=xxxxx dbname=regression
+               ->  Seq Scan on distributed_table_1513006 distributed_table
+(8 rows)
+
+EXPLAIN (COSTS OFF) INSERT INTO partitioned_distributed_table SELECT a + 1 FROM unlogged_distributed_table;
+                                         QUERY PLAN
+---------------------------------------------------------------------
+ Custom Scan (Citus INSERT ... SELECT)
+   INSERT/SELECT method: repartition
+   ->  Custom Scan (Citus Adaptive)
+         Task Count: 4
+         Tasks Shown: One of 4
+         ->  Task
+               Node: host=localhost port=xxxxx dbname=regression
+               ->  Seq Scan on unlogged_distributed_table_1513002 unlogged_distributed_table
+(8 rows)
+
+-- some aggregate queries
+SELECT sum(a) FROM partitioned_distributed_table;
+ sum
+---------------------------------------------------------------------
+  15
+(1 row)
+
+SELECT array_agg(a ORDER BY 1,2) FROM partitioned_distributed_table;
+   array_agg
+---------------------------------------------------------------------
+ {1,5,0,3,4,2}
+(1 row)
+
+SELECT array_agg(a ORDER BY 1,2) FROM unlogged_distributed_table;
```

Comment:
hmm, tests with `array_agg ORDER BY` could be flaky. Can we have another aggregate, such as
```
 select regr_syy(a,b) from partitioned_distributed_table;
```

The goal is to make sure that we can handle partial agg. without any issues